roles/quay/templates/quay.yml.j2

```diff
@@ -52,7 +52,7 @@
     superuser-password: {{ quay_password }}
     superuser-email: quay@redhat.com
 
-{% if (letsencrypt|default(true)) %}
+{% if do_quay_certs %}
```

Comment:
looks like this needs to be 
`{% if do_quay_certs == true %}`
Since this block was still executed for me when `do_quay_certs` was false

Reply:
Would you try `do_quay_certs|bool` instead of `== true`? Not because that won't work, but because I think it's messy and I hate it. :)